store/sqlstore/post_store.go

```diff
@@ -1599,7 +1608,7 @@ func (s *SqlPostStore) buildSearchPostFilterClause(fromUsers []string, excludedU
 	excludedUserClause, queryParams := s.buildSearchUserFilterClause(excludedUsers, "ExcludedUser", true, queryParams, userByUsername)
 	filterQuery = strings.Replace(filterQuery, "EXCLUDED_USER_FILTER", excludedUserClause, 1)
```

Comment:
Can we create HW tickets for this and on L1606? Would be good to get these fixed too.

Reply:
https://github.com/mattermost/mattermost-server/issues/18428